docs/testing.md

```diff
@@ -33,29 +32,20 @@ git clone https://github.com/kudobuilder/operators.git
 cd operators
 ```
 
-Make sure that you have Go version 1.12 or greater and have Go modules enabled.
+Make sure that you have the latest version of kudoctl installed.
 
-#### Start a Kubernetes cluster with kind (optional)
-
-The Operator tests rely on a functioning Kubernetes cluster. As the test harness can run KUDO, it is not necessary to install KUDO prior to running the tests.
-
-If you do not have a Kubernetes cluster to use for the tests, you can start one with [kind](https://github.com/kubernetes-sigs/kind):
+#### Run the Operator tests
 
-There is a Make target in the [Operators repository](https://github.com/kudobuilder/operators) that you can use to start it:
+To run the Operator test suite, run the following in the [Operators repository](https://github.com/kudobuilder/operators):
 
 ```
-make create-cluster
```

Comment:
@gerred I don't want these docs to be published on kudo.dev until the CLI is released again.. should I just drop it from the PR?